docs/application-connector/docs/008-details-ac-security.md

```diff
@@ -22,3 +22,18 @@ Follow these steps to disable SSL certificate verification for communication bet
   2. Edit the Deployment in Vim. Select `i` to start editing.
   3. Find the **skipVerify** parameter and change its value to `true`.
   4. Select `esc`, type `:wq`, and select `enter` to write and quit.
+
+## Override API security type
+
+The Metadata Service allows you to register APIs:
+- secured with Basic Authentication
+- secured with OAuth
+- not secured
+
```

Comment:
Shouldn't we [capitalize](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/formatting-and-style.md#ordered-and-unordered-lists) each bullet point?

Reply:
👍 done

Reply:
done